docs/release-notes/index.md

```diff
@@ -1,5 +1,54 @@
 # Release notes
 
+## ![rn_icon](https://chilipublishdocs.imgix.net/logos/CHILI_LOGOS_OK-10.svg) Apr 03, 2024 - GraFx Studio
+
+### Features
+
+#### Output settings
+
+- You can create output settings for PDF, JPG, PNG, MP4, and GIF output formats
+- There are type-specific settings to control scaling, quality, and frame rate
+- You can add a watermark to generate free-of-charge output
+- In the Template Designer workspace, you can select one of the output settings to export a template
+
+#### Print
+
+- PDF output is now available in production
+- For each layout in a template, you can specify if the intent is print, digital static, or digital animated
+- You only see the panels and properties that are relevant for the selected intent (e.g., the animation panel is only shown when the intent is digital animated)
+- You can set the unit to pixels, millimeters, or inches (this was already available in experimental mode)
+- You can set a bleed area when the intent is print (only visible on the canvas in normal view mode)
+- You can toggle between normal and preview mode with the 'W' shortcut
+- In the Studio UI, the play button and animation timeline are only shown when the intent of the selected layout is digital animated
+
+#### Other
+
+- You can change the baseline shift of text
+- You can select multiple frames and rotate them together
+- When moving or resizing multiple frames, the selection snaps to other objects
```

Comment:
This point should be removed because ticket is still in implementation and will not be included in upcoming release

Reply:
line 28 removed, not in this release